src/actions/sources/newSources.js

```diff
@@ -129,7 +129,10 @@ function checkSelectedSource(sourceId: string) {
           sourceId: source.id,
           line:
             typeof pendingLocation.line === "number" ? pendingLocation.line : 0,
-          column: pendingLocation.column
+          column:
+            typeof pendingLocation.column === "number"
+              ? pendingLocation.column
+              : 0
```

Comment:
could we do this when we set `pendingLocation.column`

Reply:
Sure!

Reply:
Hmmm, this is actually a bit difficult because this datatype is shared between both bending location selections, and pending breakpoints. Since we still want to hide the breakpoint logic behind a pref, breakpoint pending locations need to continue having an optional column in Flow at least, which means that even if we move this default setting to the creating of the pending non-breakpoint locations, we'd still need an assertion in here checking the type anyway to make sure it's always set, or something.